tests/ui/single_field_patterns.rs

```diff
@@ -0,0 +1,132 @@
+#![warn(clippy::single_field_patterns)]
```

Comment:
This should be `// run-rustfix`, since this lint is auto applicable in theory. With that comment on top of the test file, the UI tests will make sure that the suggestions are applied cleanly and don't break anything.